packages/account/src/Components/terms-of-use/terms-of-use.tsx

```diff
@@ -61,10 +63,13 @@ const TermsOfUse = ({
     onSubmit,
     value,
     real_account_signup_target,
+    residence,
     ...props
 }: TTermsOfUseProps) => {
     const { is_appstore } = React.useContext(PlatformContext);
 
+    const is_spain_residence = residence === 'es';
```

Comment:
Might be best to write hooks or helper functions for such cases and make a more generic logic out of it. 
It would make it easier for us to move this logic to V2 as well.

You can have a look at `useManualForm` as a reference.